Adjust icons according to new UI

smbea opened the pull request:
Related to #2488

- Change deploy icon
- Adjust icon sizes
- Rearrange order of app icons (right side)

Artifact
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/adjust-status-bar-icons/camunda-modeler-adjust-status-bar-icons-mac.dmg

pinussilvestrus commented on client/src/plugins/report-feedback/ReportFeedback.js:
```diff
@@ -100,7 +100,7 @@ export class ReportFeedback extends PureComponent {
     return (
       <Fragment>
-        <Fill slot="status-bar__app" group="9_report-feedback">
+        <Fill slot="status-bar__app" group="9_feedback">
```
Is there a reason for this change?

smbea replied:
We use `priority`>  `group`. The priority can only be used in the same group and the groups are sorted alphabetically. Since these icons are all in separate groups - and we can't start doing `10_....` (because 1 < 9) to order them - then we we can either put the right side icons all in the same group like `9_app-icons` and give them different priorities (in my opinion, the better way to do this). Or find a hack like renaming the group to be alphabetically first, like I did.

pinussilvestrus reviewed (approved):
Looks good to me 👍  Good job!

andreasgeier reviewed (changes requested):
Design review (see [prototype](https://fluffy-fiesta-212e1c7c.pages.github.io/prototypes/visual-ide/v15/) for reference):
- please ensure that you use all five icons from the prototype and placed in the original size of 24x24px, it seems they are cut and scaled
- icon for feedback is not correct (the expected version is different, see prototype)
- cloud connect icon has the wrong color
- on the platform button (for forms), remove the dropdown arrow
- height of the status bar should be 32px
- unread marker for what's new overlay should be blue
- increase the spacing between linting icon and linting label by 3 pixels

Please let me know if some mentioned topics are not in the scope of this PR.

smbea commented:
I addressed the points you mentioned. You can test with the updated artifact: https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/adjust-status-bar-icons/camunda-modeler-adjust-status-bar-icons-mac.dmg

> please ensure that you use all five icons from the prototype and placed in the original size of 24x24px, it seems they are cut and scaled

I ensured the viewports of all svg files (except cloud) are 0 0 24 24 and added the height and width dynamically with css to 24px. I think this solves the issue and makes them consistent but please check.

> cloud connect icon has the wrong color

This issue is dealt with the plugin. It has been changed but I believe you're still using an older package. Here is one you can test with: [cloud-connect.zip](https://github.com/camunda/camunda-modeler/files/7546602/cloud-connect-3.2.1.zip)

> increase the spacing between linting icon and linting label by 3 pixels

since I changed the svg viewport, the initial size was changed. I tried to align with the prototype but please check if it's ok.

andreasgeier reviewed (approved):
Great work @smbea - the icons are sharp and placed properly now.

_Note: I realized that I created the cloud connect icon on the wrong viewport, you mentioned it. But since you placed it already as expected, let's not do another round just because the CSS properties are not consistent._
